annotated-projects/PXD000792/PXD000792.idf.tsv

```diff
@@ -0,0 +1,34 @@
+MAGE-TAB Version	1.1
+Investigation Title	PIA - Yeast Gold Standard Benchmark Dataset
+Experiment Description	This Dataset is no actual new study but the Yeast Gold Standard benchmark dataset used in the PIA manuscript.
+
+Date of Experiment	2015-02-03
+Public Release Date	2015-05-08
+
+Protocol Name	P-MTAB-Sample-PXD000792	P-MTAB-Data-PXD000792
+Protocol Type	sample collection protocol	data analysis protocol
+Protocol Description	The sample was processed as described in Ramakrishnan, S. R. et al., Integrating shotgun proteomics and mRNA expression data to improve protein identification. Bioinformatics 2009, 25, (11), 1397-403	For these runs the RAW files were converted to mzML using the msConvertGUI and further processed to MGF files using OpenMS. For the identification a precursor tolerance of 25 ppm, a fragment tolerance of 0.5 Da, one missed cleavage and the variable modifications for oxidation of M and protein N-terminal acetylation were allowed.
+Protocol Parameters
+Protocol Hardware	LTQ Orbitrap
+Protocol Software		tandem X! Tandem Sledgehammer (2013.09.01.1);MS-GF+ Beta (v9979);PIA 1.0.0-dev-201501301530;mascot 2.4.1
+Protocol Contact
+
+Person Last Name	Uszkoreit	Marcus
+Person First Name	Julian	Katrin
+Person Mid Initials
+Person Email	julian.uszkoreit@rub.de	katrin.marcus@rub.de
+Person Phone
+Person Fax
+Person Affiliation	Ruhr-Universität Bochum,
 Medical Faculty,
 Medizinisches Proteom-Center, Medical Bioinformatics / Functional Proteomics	Ruhr-Universität Bochum Medizinisches Proteom-Center
```

Comment:
`^M` is apparently also a `newline` character as `\n`, but should it be here? The `tab` would be enough?

Reply:
Thanks, I will double check the script.

Reply:
Yes, I would replace these with spaces or remove. The validator might not complain, but if you leave this in, it may break the format, e.g. if you open the file in Excel.